make.js

```diff
@@ -1,324 +1,390 @@
+            + tabbing + tabbing + tabbing + "if (!PlayFabSettings::staticSettings->connectionString.empty())\n"
+            + tabbing + tabbing + tabbing + "{\n"
+            + tabbing + tabbing + tabbing + tabbing + "request.TitleId = PlayFabSettings::staticSettings->connectionString.substr(8,4);\n"
+            + tabbing + tabbing + tabbing + "}\n"
+            + tabbing + tabbing + "}\n"
+            + tabbing + "}\n";
+
+    if (apiCall.result === "AuthenticateIdentityResult")
+        return tabbing + "if (request.TitleId.empty())\n"
+            + tabbing + "{\n"
+            + tabbing + tabbing + "if (!settings->titleId.empty())\n"
+            + tabbing + tabbing + "{\n"
+            + tabbing + tabbing +"        request.TitleId = settings->titleId;\n"
+            + tabbing + tabbing + "}\n"
+            + tabbing + tabbing + "else\n"
+            + tabbing + tabbing + "{\n"
+            + tabbing + tabbing + tabbing + "if (!PlayFabSettings::staticSettings->connectionString.empty())\n"
+            + tabbing + tabbing + tabbing + "{\n"
+            + tabbing + tabbing + tabbing + tabbing + "request.TitleId = PlayFabSettings::staticSettings->connectionString.substr(8,4);\n"
+            + tabbing + tabbing + tabbing + "}\n"
+            + tabbing + tabbing + "}\n"
+            + tabbing + "}\n";
+            + tabbing + "if (request.PlayerAccountPoolId.empty())\n"
+            + tabbing + "{\n"
+            + tabbing + tabbing + "if (!settings->playerAccountPoolId.empty())\n"
+            + tabbing + tabbing + "{\n"
+            + tabbing + tabbing +"        request.PlayerAccountPoolId = settings->playerAccountPoolId;\n"
+            + tabbing + tabbing + "}\n"
+            + tabbing + "}\n";
+
+
+    if (apiCall.url === "/Authentication/GetEntityToken")
+        return tabbing + "std::string authKey, authValue;\n" +
+            tabbing + "if (context->entityToken.length() > 0)\n" +
+            tabbing + "{\n" +
+            tabbing + "    authKey = \"X-EntityToken\"; authValue = context->entityToken;\n" +
+            tabbing + "}\n" +
+            tabbing + "else if (context->clientSessionTicket.length() > 0)\n" +
+            tabbing + "{\n" +
+            tabbing + "    authKey = \"X-Authorization\"; authValue = context->clientSessionTicket;\n" +
+            tabbing + "}\n" +
+            "#if defined(ENABLE_PLAYFABSERVER_API) || defined(ENABLE_PLAYFABADMIN_API)\n" +
+            tabbing + "else if (settings->developerSecretKey.length() > 0)\n" +
+            tabbing + "{\n" +
+            tabbing + "    authKey = \"X-SecretKey\"; authValue = settings->developerSecretKey;\n" +
+            tabbing + "}\n" +
+            "#endif\n";
+
+    return "";
+}
+
+function getResultActions(tabbing, apiCall, isInstanceApi) {
+    if (apiCall.url === "/Authentication/GetEntityToken")
+        return tabbing + "context->HandlePlayFabLogin(\"\", \"\",  \"\", outResult.Entity->Id, outResult.Entity->Type, outResult.EntityToken);\n";

```

Comment:
I think these calls to HandlePlayFabLogin need to be updated as well. There is one less parameter in the non-Azure cases.